lib/decidim/polis/admin_engine.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Polis
+    # This is the engine that runs on the public interface of `decidim-polis`.
+    # It mostly handles rendering the created projects associated to a participatory
+    # process.
+    class AdminEngine < ::Rails::Engine
+      isolate_namespace Decidim::Polis::Admin
+
+      paths["db/migrate"] = nil
+
+      routes do
+        resources :polis do
+          resources :attachments
+        end
+
+        root to: redirect('../edit')
```

Comment:
`root to: redirect('../edit')` il n'existe pas une vraie route que tu pourrais utiliser à ce niveau ? Je pense que comme tu définis `resources :polis` au dessus, tu pourrais ensuite utiliser la route edit au lieu du redirect(../edit). C'est une supposition rien de sûr.